src/diagnostics/default_diagnostics.jl

```diff
@@ -329,20 +329,23 @@ function default_diagnostics(
             "gpp",
             "ct",
             "lai",
-            "sco2",
```

Comment:
this is mostly NaN right now anyways... we can add it back later when we have time to debug it